swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+import copy
+
+# Wave 1 
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = copy.copy(inventory)
```

Comment:
I like how you make a copy of the inventory here! This is a good defensive programming practice that prevents others from messing around with our inventory inadvertently after passing it to us.